components/css/button.css

```diff
@@ -1,36 +1,34 @@
 .btn {
-  border-radius: 4px;
-  color: var(--btn-primary-text);
+  border-radius: var(--size-1);
   transition: 0.3s;
   opacity: 1;
   display: flex;
   justify-content: center;
   align-items: center;
+  color: inherit;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  outline: none;
```

Comment:
It's not good to do outline `none`. 
when we use `TAB` to focus on buttons then the outline shows, without it, the user will not get any visible feedback if the button is in focus or not.